osu.Game.Rulesets.Tau/Skinning/Legacy/TauLegacySkinTransformer.cs

```diff
@@ -27,8 +27,8 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
                 case TauSkinComponents.Handle:
                     return Source.GetTexture("handle") != null ? new LegacyHandle() : null;
 
-                case TauSkinComponents.Playfield:
-                    return Source.GetTexture("field-overlay") != null ? new LegacyPlayfield() : null;
+                case TauSkinComponents.Ring:
+                    return Source.GetTexture("ring-overlay") != null ? new LegacyPlayfield() : null;
```

Comment:
You changed the texture that is used to determine whether `LegacyPlayfield` is used, but `LegacyPlayfield` still uses the previous lookup.

https://github.com/Altenhh/tau/blob/master/osu.Game.Rulesets.Tau/Skinning/Legacy/LegacyPlayfield.cs#L30